_posts/2015-04-01-aurelia-io.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Aurelia.io
+author: arto
+excerpt: Coding with Aurelia JavaScript framework and ECMAScript 6 - right now
+---
+I've always been fond of keeping my eyes at the horizon, constantly watching what new technologies will emerge, what useful tools will we have in the years to come. With this mindset I recently started a quest to learn Angular 2 - before it will be mainstream. I had studied it a bit earlier, too, but then it felt way too ugly and early to have anything to do with project work. Working in [Solita](http://www.solita.fi) - there is no excuse to let your skills or knowledge go stagnant - or letting your toolbox to be outdated. So I went looking for Angular 2 - but ended up finding something else.
```

Comment:
pieniä ehdotuksia idiomaattisuuden parantamiseksi tässä alussa:

keeping _an eye on_ the horizon, _to watch_ what new technologies will emerge _and_ what useful tools _we will have in the next few years_

"work in" huonompi yrityksen kanssa kuin vaihtoehdot:  "As a guide, you work for an employer, in a department, at a location. For example, you could work for Airbus, in the finance department, at the research facility."  http://english.stackexchange.com/questions/86751/to-work-for-vs-to-work-in-vs-to-work-at

tässä kappaleessa ajatusviivoja - voisi vähentää. Piste, pilkku tai ei välimerkkiä toimisi monessa kohtaa yhtä hyvin tai paremmin.